buildlib/tools/builds.sh

```diff
@@ -349,6 +349,7 @@ prepare() {
 	az_init_modules
 	env
 	cd ${WORKSPACE}
+	make_clean distclean || true
```

Comment:
why not fail in case of error?
also, will it clean "ucx-v1.11" folder?
must make sure no leftovers after the previous build